launchers/director_serve.go

```diff
@@ -45,7 +45,7 @@ func DirectorServe(ctx context.Context, engine *gin.Engine, egrp *errgroup.Group
 
 	director.LaunchMapMetrics(ctx, egrp)
 
-	director.LaunchServerCountMetric(ctx, egrp)
+	director.LaunchServerCountMetric()
```

Comment:
Since this isn't actually launching a go routine anymore, I'm not sure I love keeping the setup here (which mostly deals with launching background tasks).

Maybe move this closer to where the `serverAds` cache is actually constructed? This may be a good time to use an `init()` function in the `director` package.